ui/src/sandbox.rs

```diff
@@ -651,6 +673,7 @@ pub struct ExecuteRequest {
     pub tests: bool,
     pub backtrace: bool,
     pub code: String,
+    pub stdin: Option<String>,
```

Comment:
Would it make sense to drop the `Option` — is there any conceptual difference between passing in an empty string?

Reply:
Shouldn't make much of a difference other than the fact that `stdin` is optional in the request payload.